.gitignore

```diff
@@ -13,3 +13,4 @@ functional-output/
 dist
 submit-your-appeal.iml
 mochawesome-report/
+yarn.lock
```

Comment:
shouldn't ignore this. The whole point of yarn.lock is that we all have the same version